dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/AndroidTracer.kt

```diff
@@ -137,5 +139,29 @@ class AndroidTracer internal constructor(
         internal const val TRACE_LOGGER_NAME = "trace"
 
         internal const val TRACE_ID_BIT_SIZE = 63
+
+        /**
+         * Helper method to attach a Throwable to a specific Span.
+         * The Throwable information (Class name, message and Stacktrace) will be added to the
+         * provided Span as special Tags.
+         * @param forSpan as the active Span
+         * @param throwable as the Throwable you wan to log
+         */
+        @JvmStatic
+        fun logThrowable(forSpan: Span, throwable: Throwable) {
+            val fieldsMap = mutableMapOf(Fields.ERROR_OBJECT to throwable)
+            forSpan.log(fieldsMap)
+        }
+
+        /**
+         * Helper method to attach an error message to a specific Span.
+         * @param forSpan as the active Span
+         * @param message as the error message you want to attach
```

Comment:
```suggestion
         * Helper method to attach an error message to a specific Span.
```suggestion
         * The error message will be added to the provided Span as a standard Error Tag.
         * @param span the active Span
         * @param message the error message you want to attach
```